README.md

```diff
@@ -328,7 +329,8 @@ sh.setup(
     device_id=1,
     grpc_addr='localhost:9559',
     election_id=(0, 1), # (high, low)
-    config=sh.FwdPipeConfig('config/p4info.pb.txt', 'config/device_config.bin')
+    config=sh.FwdPipeConfig('config/p4info.pb.txt', 'config/device_config.bin'),
+    role_name="controller_1"
```

Comment:
I suggest omitting this until we have a released version of P4Runtime (v1.4) that supports string roles. This way we can merge this PR now